client/my-sites/importer/section-import.jsx

```diff
@@ -307,7 +315,17 @@ class SectionImport extends Component {
 					{ isJetpack && ! isAtomic && ! isEnabled( 'importer/unified' ) ? (
 						<JetpackImporter />
 					) : (
-						this.renderImportersList()
+						<>
+							{ /** Show a plugin update warning if Jetpack version does not support import endpoints */ }
+							{ isJetpack && ! isAtomic && (
+								<JetpackPluginUpdateWarning
+									siteId={ this.props.siteId }
+									minJetpackVersion={ JETPACK_IMPORT_MIN_PLUGIN_VERSION }
+									warningRequirement={ translate( 'To make sure you can import reliably' ) }
+								/>
+							) }
+							{ this.renderImportersList() }
```

Comment:
Should we block the actions you can take when you click an element of the importer's list? Or maybe show them in an inactive state?

Reply:
Have updated! Please re-review.